waiter/dashboards/views.py

```diff
@@ -1,10 +1,16 @@
 from datetime import datetime, timedelta
+from django.conf import settings
 
 from django.views.generic.base import TemplateView
 from django.utils.safestring import mark_safe
 
 from runs.models import *
 
+import redis
+REDIS = redis.StrictRedis(host=settings.MMVP_REDIS_HOST,
+                          port=settings.MMVP_REDIS_PORT,
+                          db=settings.MMVP_REDIS_DB)
```

Comment:
You'll need to set encoding + decode options here --- otherwise redis returns raw byte strings.